addons/sale/tests/test_sale_to_invoice.py

```diff
@@ -922,7 +922,11 @@ def test_partial_invoicing_interaction_with_invoicing_switch_threshold(self):
 
         self.assertEqual(line.qty_invoiced, 10)
         line.qty_delivered = 15
-        self.assertEqual(line.qty_invoiced, 10)
+        invoice = sale_order._create_invoices()
+        invoice.action_post()
+
+        self.assertEqual(line.qty_invoiced, 15)
+        self.assertEqual(sale_order.order_line.untaxed_amount_invoiced, line.qty_invoiced * line.price_unit)
```

Comment:
Is there a need to create a second invoice? It seems to me that the problem would already occur without creating a new invoice.
```suggestion
        self.assertEqual(sale_order.order_line.untaxed_amount_invoiced, 700.0)
```
I think it is better to put the number explicitly. Maybe we could explicitly put the `price_unit` (70) on the order line to make it clearer where the 700 comes from?